docs/mkdocs.yml

```
@@ -0,0 +1,94 @@
+# yaml-language-server: $schema=https://squidfunk.github.io/mkdocs-material/schema.json
+
+site_name: Event Sourcing
```

```suggestion
site_name: Event Sourcing Bundle
```
Add Bundle here maybe?